### PR TITLE
Fix README example

### PR DESCRIPTION
**Pull request by LegNeato**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Or expanding an abbreviation in text:
 
 ```rust
 let parser = parser.map(|event| match event {
-	Event::Str(text) => Event::Str(text.replace("abbr", "abbreviation")),
+	Event::Text(text) => Event::Str(text.replace("abbr", "abbreviation")),
```
Comment (raphlinus) [inline README.md]: Shouldn't the `Event::Str` on the RHS also be changed?

Comment (LegNeato) [reply README.md]: Whoops, fixed!

**Comment by raphlinus** [comment]
Any special reason you closed this? Sorry about not responding earlier, but I think it's mergeable...

**Comment by LegNeato** [comment]
Nope, was just clearing out old PRs :-)

**Comment by raphlinus** [comment]
Thanks!

**Pull request merged**